app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -389,18 +389,21 @@ private BitmapDrawable decodeBlurHash(String blurhash) {
         return ImageLoadingHelper.decodeBlurHash(this.avatar.getContext(), blurhash);
     }
 
-    private void loadImage(MediaPreviewImageView imageView, String previewUrl, MetaData meta,
+    private void loadImage(MediaPreviewImageView imageView,
+                           @Nullable String previewUrl,
+                           @Nullable MetaData meta,
                            @Nullable String blurhash) {
+
         Drawable placeholder = blurhash != null ? decodeBlurHash(blurhash) : mediaPreviewUnloaded;
+
         if (TextUtils.isEmpty(previewUrl)) {
-            if (blurhash != null) {
```

Comment:
I wanted to say that it's unlikely that there's no preview but there's a blurhash but then Pleroma doesn't have previews. I'm not sure if they have blurhashes though.